### DISCUSSION TITLE
How to allow anyone to approve changes on a PR

### DISCUSSION THREAD

**Discussion by okseb** [Mergifyio/mergify#5096] [Q&A] [answered]
So I have the basic setup that Mergify starts with [here](https://github.com/rowbawts/theopenestsource/blob/5de51dbf58d7e29b702e57ec3c1bd43996f450a3/.mergify.yml) and when someone approves the changes as seen [here](https://github.com/rowbawts/theopenestsource/pull/3) the status still remains as if nobody has reviewed it (visible [here](https://github.com/rowbawts/theopenestsource/pull/3/checks?check_run_id=16155676627)) is there a way to resolve this?

**Comment by DouglasBlackwood** [comment, marked as answer]
It seems that the user who approved the pull request doesn't have write access to the repository (according to the gray check).

**Reply by okseb** [reply]
That's what I guessed but that's on purpose as I was trying to see if I could get it setup where anybody that approves a PR regardless of permission can be detected, Is this unsupported behavior? As I understand normally this would never be wanted.

**Reply by jd** [reply]
Yes, this is not supported. I don't think that's something anyone would want?

**Reply by okseb** [reply]
Yeah makes sense I just wanted to do an experiment where I was going to raise the amount of approvals required to get Mergify to merge a PR so the community can all "vote" on what to merge and allow them to federate a repo amongst themselves rather than the repo admins. And I didn't want to have to write my own bot to do it if I could find something that would work.
